2/contrib/openshift/base-plugins.txt

```diff
@@ -18,30 +18,30 @@ openshift-sync:1.0.2
 # processed sec adv https://jenkins.io/blog/2017/07/10/security-advisory/
 # processed sec adv https://jenkins.io/security/advisory/2017-08-07/
 #
-config-file-provider:2.16.2
-docker-commons:1.8
-job-dsl:1.60
-parameterized-trigger:2.35.1
-pipeline-build-step:2.5.1
+config-file-provider:2.17
+docker-commons:1.11
+job-dsl:1.68
+parameterized-trigger:2.35.2
+pipeline-build-step:2.7
 pipeline-input-step:2.8
-script-security:1.39
+script-security:1.41
 
 # Legacy stuff
-mapdb-api:1.0.1.0
-subversion:2.9
+mapdb-api:1.0.9.0
+subversion:2.10.2
 
 # remote loader https://wiki.jenkins-ci.org/display/JENKINS/Pipeline+Remote+Loader+Plugin
-workflow-remote-loader:1.2
+workflow-remote-loader:1.4
 
 # mercurial - https://wiki.jenkins-ci.org/display/JENKINS/Mercurial+Plugin
-mercurial:1.54
-matrix-project:1.7.1
+mercurial:2.2
+matrix-project:1.12
 ssh-credentials:1.13
 # pulling in 1.13 for https://issues.jenkins-ci.org/browse/JENKINS-41760, but this is also a dependency
 # of pipeline-model-definition (declaritive pipeline) pulls it in directly, as well as indirectly via its
 # dependency on docker-workflow/docker-common.  We should monitor when that dependency is
 # bumped to 1.13 or later, and then remove this line
-credentials-binding:1.13
+credentials-binding:1.15
```

Comment:
This one is part of the latest security advisory, so it would be good to see this get in at least! :+1: https://jenkins.io/security/advisory/2018-02-05/

Reply:
Per the comment above, I'll take a peek at the references to credential-binding, and see if 1.15 was referenced elsewhere ... probably not, but if we can remove citing this plugin explicitly I'd like to.

Reply:
Also, per @grdryn note, @firemanxbr - could you please add a comment right before credentials-binding that cites the https://jenkins.io/security/advisory/2018-02-05/ security advisory ... as you can see above, It helps us to track in some fashion when we've addressed those specifically

Reply:
Yeah none of the other explicitly listed plugins bring in 1.15 of credentials-binding ... still only 1.11 or 1.12